python/tvm/topi/x86/dense.py

```diff
@@ -373,6 +375,153 @@ def _callback(op):
     return s
 
 
+def dense_amx_int8_compute(cfg, data, packed_w, bias=None):
```

Comment:
This can be shared with the vnni compute.

Reply:
already unified AMX and VNNI to `dense_int8`